src/java.desktop/macosx/classes/sun/java2d/macos/MacOSFlags.java

```diff
@@ -44,28 +52,40 @@
 
     private static native boolean initNativeFlags();
 
-    private static boolean getBooleanProp(String p, boolean defaultVal) {
+    private static PropertyState getBooleanProp(String p, PropertyState defaultVal) {
         String propString = System.getProperty(p);
-        boolean returnVal = defaultVal;
+        PropertyState returnVal = defaultVal;
```

Comment:
I guess all these get/setProperty calls are used only from that doPrivileged block in initJavaFlags() ?